cypress/page-objects/preview.js

```diff
@@ -1,79 +1,80 @@
+    getMobPreview() {
+        return cy.get( 'div.wp-touch-popup' ).find( 'div.wikipediapreview' )
     }
 
-
-    // Wikipedia Preiew Header
-
-    Header () {
-        return cy.get('div.wp-popup  > div.wikipediapreview > div.wikipediapreview-header')
+    getMobBackgroundScreen() {
+        return cy.get( '.wp-dark-screen')
     }
+	// Wikipedia Preiew Header
 
-    HeaderImage () {
-        return cy.get('div.wp-popup  > div.wikipediapreview > div.wikipediapreview-header').find('div.wikipediapreview-header-image')
-    }
+	getHeader() {
+		return cy.get( 'div.wikipediapreview' ).find( 'div.wikipediapreview-header' )
+	}
 
-    HeaderWatermark () {
-        return cy.get('div.wp-popup  > div.wikipediapreview > div.wikipediapreview-header').find('div.wikipediapreview-header-wordmark')
-    }
+	getHeaderImage() {
+		return cy.get( 'div.wikipediapreview' ).find( 'div.wikipediapreview-header-image' )
+	}
 
-    HeaderClosebtn () {
-        return cy.get('div.wp-popup  > div.wikipediapreview > div.wikipediapreview-header').find('div[class="wikipediapreview-header-closebtn"]')
-    }
+	getHeaderWatermark() {
+		return cy.get( 'div.wikipediapreview' ).find( 'div.wikipediapreview-header-wordmark' )
+	}
 
+	getHeaderClosebtn() {
+		return cy.get( 'div.wikipediapreview' ).find( 'div[class="wikipediapreview-header-closebtn"]' )
+	}
 
+	// Wikipedia Preiew Body
 
+	getBody() {
+		return cy.get( 'div.wikipediapreview' ).find( 'div.wikipediapreview-body' )
+	}
 
-    // Wikipedia Preiew Body
+	getBodyContent() {
+		return cy.get( 'div.wikipediapreview' ).find( 'div.wikipediapreview-body > p' )
+	}
 
-    Body () {
-        return cy.get('div.wp-popup > div.wikipediapreview > div.wikipediapreview-body')
-    }
+	getBodyGallery() {
+		return cy.get( 'div.wikipediapreview' ).find( 'div.wikipediapreview-gallery-row' )
+	}
 
-    BodyContent () {
-        return cy.get('div.wp-popup > div.wikipediapreview > div.wikipediapreview-body > p')
-    }
-
-    BodyGallery () {
-        return cy.get('div.wp-popup > div.wikipediapreview > div.wikipediapreview-body > div.wikipediapreview-gallery')
-    }
-
-    BodyMessage () {
-        return cy.get('div.wp-popup > div.wikipediapreview > div.wikipediapreview-body').find('div.wikipediapreview-body-message')
-    }
-
-    BodyAction () {
-        return cy.get('div.wp-popup > div.wikipediapreview > div.wikipediapreview-body').find('div.wikipediapreview-body-action > a')
+    getBodyGalleryImages() {
+        return cy.get( 'div.wikipediapreview' ).find( 'div.wikipediapreview-gallery-image' )
     }
 
+	getBodyMessage() {
+		return cy.get( 'div.wikipediapreview' ).find( 'div.wikipediapreview-body-message' )
+	}
 
+	getBodyAction() {
+		return cy.get( 'div.wikipediapreview' ).find( 'div.wikipediapreview-body-action > a' )
+	}
 
+	//  Wikipedia Preview Footer
 
-    //  Wikipedia Preview Footer
+	getFooter() {
+		return cy.get( 'div.wikipediapreview' ).find( ' div.wikipediapreview-footer' )
```

Comment:
why not do 
`cy.get('div.wikipediapreview-footer')` 

only instead of get+find?